packages/geoprocessing/src/toolbox/overlapRaster.ts

```diff
@@ -20,7 +20,9 @@ export async function overlapRaster(
   /** single sketch or collection to calculate metrics for. */
   sketch:
     | Sketch<Polygon | MultiPolygon>
-    | SketchCollection<Polygon | MultiPolygon>
+    | SketchCollection<Polygon | MultiPolygon>,
+  /** Simplifies results to 6 significant digits to avoid floating pt arithmetric differences, defaults to false */
+  simplifyPrecision?: boolean
```

Comment:
I would suggest that this need to be within an options object

Reply:
Updated with the following parameter
`options?: Partial<OverlapRasterOptions>`

```
interface OverlapRasterOptions {
  /** Truncates results to 6 digits, defaults to false */
  truncate?: boolean;
}
```